### ISSUE TITLE
Regresssion (?): Cannot use collection property name that is the same as a function name

### ISSUE THREAD

**Issue by LeaVerou** [mavoweb/mavo#828]
Testcase: https://codepen.io/leaverou/pen/wvPrOqm?editors=1100

I think this is a regression, @DmitrySharabin can you verify?

IIRC we have a test about using property names that conflict with function names, but not collection property names.

**Comment by DmitrySharabin** [comment]
> I think this is a regression

You are right, it's a regression. The first bad commit is the one that added `mv-list/mv-list-item` to Mavo. 😔
3410cb8777bdd2851d1b72b21eaa352a18f0b30e

> IIRC we have a test about using property names that conflict with function names, but not collection property names.

The only thing I could find is this (but it's about a function that clashes with prototype function): https://test.mavo.io/mavoscript.html#function-rewrite
